docs/contributing/coding-style.md

```diff
@@ -603,3 +627,17 @@ We do not suggest to modify the generated code other than the `tests` variable,
         }
         // generated test code
     ```
+
+1. Unused fields
+
+    By default, the template provides `fields` structure to initialize object of the test target. But in some cases, not all `fields` are needed. So please delete unnecessary fields.
+
+    If the test case needs only `addr` of `fields`, please delete other fields.
+
```

Comment:
I think it is better to provide the example of the struct and the corresponding test code.